src/pages/about-us/history.tsx

```diff
@@ -23,14 +35,64 @@ export default function HistoryPage({
   if (!sanityHistoryPage)
     throw `No Sanity document for the culture page was found.`
 
+  // for later:
+  // const theme = useTheme()
+  // const matches = useMediaQuery(theme.breakpoints.up("md"))
+
+  const { timeline } = sanityHistoryPage
+
+  const renderTimeline = () => {
```

Comment:
Ooo I've never seen the renderTimeline() function way of doing this. I think I would've created a separate Timeline component since its complex enough here to justify doing that.

```
// Defined outside the main SanityHistoryPage component
const Timeline = ({timeline: SomeGraphQLTypeOrWhatever}) => (
  <Stack>
     ...
  <Stack>
)
```

And then if you wanted you could do 

```
{timeline && <Timeline timeline={timeline />}
```

Which I guess is a bit ugly with all those repeated names. Alternatively, you could've in-lined it using JSX with
```
// Defined in-line here
const Timeline = timeline ? (
 <Stack>
    ...
 </Stack>
) 
: <></>
```